README.md

```diff
@@ -379,12 +385,12 @@ A Batcher with a rate limiter depends on each operation having a cost. The follo
 
 ## Areas for improvement
 
-- There is currently no way to limit concurrency, but there should be. This could be implemented such that no more than "x" operations are flight at any given time.
-
 - This tool was originally designed to limit transactions against Azure Cosmos which has a cost model expressed as a single composite value (Request Unit). For datastores that might have more granular capacities, it would be nice to be able to provision Batcher with all those capacities and have an enqueue method that supported those costs. For example, memory, CPU, disk, network, etc. might all have separate capacities and individual queries might have individual costs.
 
 - There is currently no way to change capacity in the rate limiters once they are provisioned, but there is no identified use-case yet for this feature.
 
 - There is currently no good way to model a datastore that autoscales but might require some time to increase capacity. Ideally something that allowed for capacity to increase by "no more than x amount over y time" would helpful. This could be a rate limiter or a feature that is added to existing rate limiters.
 
 - The pause logic is a simple fixed amount of time to delay new batches, but it might be nice to have an exponential back-off.
+
+- Currently the only shared capacity rate limiter is for Azure. It would be nice to add support for Zookeeper, Consul, or etcd.
```

Comment:
Another option worth mentioning here might be redis/redsync

Reply:
Added.